package.json

```diff
@@ -800,6 +804,11 @@
             "scope": "window",
             "default": false,
             "markdownDescription": "If set to `true` enables Stateful Authentication Provider"
+          },
+          "runme.foyleAddress": {
```

Comment:
I'm wondering if we should have a feature flag for Foyle for the time being. We do have a simple abstraction that has served us well:

https://github.com/stateful/vscode-runme/blob/main/src/extension/kernel.ts#L179-L181

Reply:
> What would the experiment flag control? Is there a way to gate the configuration options based on the experiment? I think an experiment flag might be redundant with foyleAddress. I lean towards using a non-empty value of foyleAddress a good way to signal foyle is enabled. I think if we tell users they need to do two things - 1) set a non-empty address 2) flip the experiment - it increases the likelihood of mistakes due to user error; i.e. user only sets one of them and wonders why it isn't working.

non-empty `foyleAddress` would work sufficiently. the experiment/feature flag would 1.) allow explicit opt-it and 2.) special case anywhere in the extension code accordingly. Specifically for logging detailed execution data we'd like to disable persistent logs for any unsuspecting Runme user. I agree we can defer making those change as part of resolving Foyle-compatible logging.